src/controllers/roleControllers.ts

```diff
@@ -0,0 +1,85 @@
+import logger from '../logs/config';
+import { Request, Response } from 'express';
+import Role from '../database/models/role';
+
+const createRole = async (req: Request, res: Response): Promise<void> => {
+  try {
+    const { name, displayName } = req.body;
+    const createdRole = await Role.create({ name, displayName });
+    res.status(201).json({ ok: true, data: createdRole });
+  } catch (error) {
+    logger.error(error);
+    console.log(error);
+    res.status(400).json({
+      ok: false,
+      errorMessage: 'Role could not be created successfully',
+      error,
+    });
```

Comment:
Please use the send `sendInternalErrorResponse` function here, check how it was done in the login.